Chaitra/arrayy/3.c

```diff
@@ -0,0 +1,28 @@
+/* Program to find duplicate elements inan array */
+
+#include <stdio.h>
+int main()
+{
+    int arr[100]; i, j, n, count = 0;
```

Comment:
i is not declared properly. Giving compiler error.

Reply:
ok i will check it